### PR TITLE
Update query return value and add get_dataframe method

### PR DESCRIPTION
**Pull request by mishaaskerka**
Changes:
- Fixed up the README 
- json response from the query method now has rows and columns
- added a convenience class method
- added pandas as a new requirement
- upped Python to 3.7

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -10,9 +10,10 @@ repository = "https://github.com/Ben-Hu/metabasic"
 include = ["metabasic/py.typed"]
 
 [tool.poetry.dependencies]
-python = ">=3.6"
+python = ">=3.7"
```
Comment (Ben-Hu) [inline pyproject.toml]: `
```suggestion
python = ">=3.6.1"
```

Should be sufficient here to satisfy the Pandas python version requirement while keeping compatibility with python 3.6

Comment (mishaaskerka) [reply pyproject.toml]: done!

**Comment by Ben-Hu** [comment]
mypy is complaining about missing type stubs for pandas
let's silence those for now by adding the following to mypy.ini

```
[mypy-pandas.*]
ignore_missing_imports = True
```

File: metabasic/metabasic.py [hunk 2]
```diff
@@ -5,6 +5,8 @@
 
 from .exceptions import AuthError, ConfigError
 
+import pandas as pd
```
Comment (Ben-Hu) [inline metabasic/metabasic.py]: this should be grouped with the external dependency inquirer/requests imports re: [pep8](https://www.python.org/dev/peps/pep-0008/#id23) 

isort should handle this for you, code formatting can be run via the `make format` target

**Pull request merged**